src/lib/styles/global/modal.scss

```diff
@@ -5,19 +5,12 @@
  */
 
 div.modal {
-  .wizard-wrapper {
+  form {
```

Comment:
Isn't this dangerous?

Any form inside the modal will behave the same way. We wanted it or not...

Reply:
I thought about it too - i.e. using a style class - but at the end of the day, we indeed probably want all form to look the same.

Not dangerous in any case because it only has to do with new design.

Reply:
Us yes, but not necessarily all the users of gix-components might want that.